CODEOWNERS

```
@@ -214,6 +214,7 @@ k8s/perftest/common/am/accessmgmt-api.yaml @hmcts/am-devs
 k8s/**/common/ethos/namespace.yaml @hmcts/ethos
 k8s/**/common/ethos/identity.yaml @hmcts/ethos
 k8s/**/common/ethos/repl-docmosis-service.yaml @hmcts/ethos
+k8s/**/common/ethos/ecm-consumer.yaml @hmcts/ethos
```

if it's not allowed in prod then this shouldn't be **

```suggestion
k8s/aat/common/ethos/ecm-consumer.yaml @hmcts/ethos
```

---

I deleted the yaml file from prod. It should not be found anymore.

---

> if it's not allowed in prod then this shouldn't be **

 deleted the yaml file from prod. It should not be found anymore.